index.html

```diff
@@ -16,6 +16,10 @@ <h1>Mirco Warnck</h1>
       <a href="#biography">Biography</a>
     </nav>
   </header>
+  <figure>
+  <img src="https://source.unsplash.com/random/600x300" alt="Random Bild">
```

Comment:
I think that "Random Bild" is not the best way of combining English and German here ;)